modules/ROOT/attachments/admin-api.yaml

```diff
@@ -4831,10 +5102,12 @@ tags:
       Manage partitions of a Redpanda cluster.
 
   - name: Tiered Storage
-    description: Manage
+    description: Manage tiered storage.
```

Comment:
Given that it is an Enterprise licensed feature, this should be "Manage Tiered Storage."